commandapi-platforms/commandapi-bukkit/commandapi-bukkit-core/src/main/java/dev/jorel/commandapi/PaperImplementations.java

```diff
@@ -36,8 +44,24 @@ public PaperImplementations(boolean isPaperPresent, boolean isFoliaPresent, NMS<
 		} catch (ClassNotFoundException e) {
 			// uhh...
 		}
-		
 		this.feedbackForwardingCommandSender = tempFeedbackForwardingCommandSender;
+
+        this.paperCommandSendingPool = getThreadPoolExecutor(nmsInstance);
+	}
+
+	@Nullable
+	private static ThreadPoolExecutor getThreadPoolExecutor(NMS<?> nmsInstance) {
+		Class<?> nmsCommandsClass = nmsInstance.getNMSCommandsClass();
+		try {
+			// public static field
+			Field commandSendingPoolField = nmsCommandsClass.getField("COMMAND_SENDING_POOL");
```

Comment:
This temporary variable is declared and then immediately used. We don't need to create this temporary variable here.